CHANGELOG.md

```diff
@@ -32,6 +32,7 @@ This release will bump the Realm file format from version 22 to 23. Opening a fi
 * `RealmUUID.random()` would generate the same values when an app was re-launched from Android Studio during development. (Issue [#1123](https://github.com/realm/realm-kotlin/pull/1123)) 
 * Complete flows with an IllegalStateException instead of crashing when notifications cannot be delivered due to insufficient channel capacity (Issue [#1147](https://github.com/realm/realm-kotlin/issues/1147))
 * [Sync] Custom loggers now correctly see both normal and sync events. Before, sync events were just logged directly to LogCat/StdOut.
+* [Sync] When a `SyncSession` was paused using `SyncSession.pause()`, it would sometimes automatically resume the session. (Core Issue [#6085](https://github.com/realm/realm-core/issues/6085))
```

Comment:
Shouldn't we somehow mention the new `State.PAUSED. I would deem it ok to add as a side effect of the fix when  it also goes into a minor release, but would expect it to be mentioned here for visibility.